_pydevd_bundle/pydevd_frame_utils.py

```diff
@@ -34,7 +34,9 @@ def add_exception_to_frame(frame, exception_info):
 
 
 def remove_exception_from_frame(frame):
-    frame.f_locals.pop("__exception__", None)
+    # In 3.13 frame.f_locals became a proxy for a dict, It does not
+    # have methods to allow items to be removed, only added. So just set the item to None.
+    frame.f_locals["__exception__"] = None
```

Comment:
Not super happy with this. We can no longer remove things from f_locals in a frame. This was the best I could do, but maybe we need to special case variables response to skip this item when it's None?

Reply:
This is a 3.13 thing (which according to the following merged PR: https://github.com/python/cpython/pull/125616 will probably be backported to 3.13.1 -- it's really a regression).

I did a change in https://github.com/fabioz/PyDev.Debugger/commit/e844e4283341111d719bfd35fd5f0298e868891f which addresses this only on python 3.13.0